src/ucp/wireup/wireup_ep.c

```diff
@@ -465,29 +468,25 @@ static UCS_CLASS_CLEANUP_FUNC(ucp_wireup_ep_t)
     ucs_debug("ep %p: destroy wireup ep %p", ucp_ep, self);
 
     uct_worker_progress_unregister_safe(worker->uct, &self->progress_id);
+
     if (self->aux_ep != NULL) {
-        ucs_queue_head_init(&tmp_pending_queue);
-        /* Discard AUX UCT EP to purge all outstanding/pending operations.
-         * Normally, WIREUP EP should complete all outstanding operations prior
-         * destroying WIREUP EP - so, doing flush(CANCEL) won't take any affect,
-         * but it will make sure that no completions will be received if some
-         * error was detected */
-        ucp_wireup_ep_discard_aux_ep(self, UCT_FLUSH_FLAG_CANCEL,
-                                     ucp_request_purge_enqueue_cb,
-                                     &tmp_pending_queue);
-        self->aux_ep = NULL;
-        ucp_wireup_replay_pending_requests(ucp_ep, &tmp_pending_queue);
+        uct_eps[i]     = self->aux_ep;
+        rsc_index[i++] = self->aux_rsc_index;
     }
-
     if (self->super.is_owner && (self->super.uct_ep != NULL)) {
-        ucp_worker_discard_uct_ep(self->super.ucp_ep, self->super.uct_ep,
-                                     self->super.rsc_index,
-                                     UCT_FLUSH_FLAG_CANCEL,
-                                     ucp_destroyed_ep_pending_purge,
-                                     self->super.ucp_ep,
-                                     (ucp_send_nbx_callback_t)ucs_empty_function,
-                                     NULL);
-        ucp_proxy_ep_set_uct_ep(&self->super, NULL, 0, UCP_NULL_RESOURCE);
+        uct_eps[i]   = self->super.uct_ep;
+        rsc_index[i] = self->super.rsc_index;
+    }
+    for (i = 0; uct_eps[i] != NULL; i++) {
```

Comment:
per https://github.com/openucx/ucx/pull/7819#discussion_r793487846, this looks worse now - better revert and just fix indentation

Reply:
done